jupiterone/questions/questions.yaml

```diff
@@ -1,4 +1,18 @@
 ---
-sourceId: managed:template
+sourceId: managed:terraform
 integrationDefinitionId: '${integration_definition_id}'
-questions: []
\ No newline at end of file
+questions: 
+- id: integration-question-terraform-cloud-users-with-no-multi-factor-authentication
+  title: Are there any Terraform users that do not use MFA in a Terraform team workspace or as an individual contributor?
```

Comment:
The title claims that we are searching for users who do not have MFA enabled, but the query does not consider any MFA filters. All users will be returned from the query. Can we please supply a filter that makes sense here?

Reply:
I'll leave that to you @mcpeach9. Good catch @austinkelleher I was def on autopilot on the contents of question 😬